src/server/models/error.ts

```diff
@@ -0,0 +1,21 @@
+export class ErrorWrapper {
+  public message: string;
+  public status?: number;
+  public code?: string;
+  public authErrorCode?: string;
+  public requestToken?: string;
+
+  constructor(
+    message?: string,
+    status?: number,
+    code?: string,
+    authErrorCode?: string,
+    requestToken?: string
+  ) {
+    this.message = message;
+    this.status = status || 500;
+    this.code = code || '';
+    this.authErrorCode = authErrorCode;
```

Comment:
authErrorCode ?

Reply:
Not of any use. As we have error code.. if required some will add the same for project specific.

Reply:
please remove it as it is not required